src/utils/bignumber.ts

```diff
@@ -0,0 +1,5 @@
+import BigNumber from 'bignumber.js';
+
+BigNumber.config({ DECIMAL_PLACES: 100, EXPONENTIAL_AT: 1e9 });
```

Comment:
We don't really need [`BigNumber libraries anymore`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)—Unless we really need decimal operations.

Reply:
Fixed in 6cd632d7c520d20ea5f02a83b1ea1b4da83ad696